dockerizeit/master/permissions.properties

```diff
@@ -0,0 +1,12 @@
+user1 {
+		userId = hudson.security.ACL.ANONYMOUS_USERNAME
+		permissions = [
+			"hudson.model.Computer.Create",
```

Comment:
I'm not sure to understand what to do you mean import  'import hudson.security.Permission' and 'import hudson.model.Computer'  in properties file?